modules/formHandler.js

```diff
@@ -0,0 +1,26 @@
+import createBookList, { booksContainer } from './createBookList.js';
+import { updateBooks } from './getLocalStorage.js';
+import saveToLocalStorage from './setLocalStorage.js';
+
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const books = updateBooks;
+
+const resetForm = () => {
+  title.value = '';
+  author.value = '';
+};
+
+const formSubmitHandler = () => {
+  books.push({
+    title: title.value,
+    author: author.value,
+    id: `${Math.random().toString(36).substr(2, 9)}`,
+  });
+
+  booksContainer.innerHTML = '';
+  createBookList(books);
+  resetForm();
+  saveToLocalStorage(books);
+};
```

Comment:
- [ ] It is required to ensure that application is working as expected. It should be possible to add and remove books from the list. Below is the screenshot of what I got while trying to add book to the list.

### Your App
![addBook](https://github.com/Yonashenok/Awesome-Books-ES6/assets/26597432/83de96c6-f3b3-4037-a190-7febe62a06bc)